core/src/main/java/hudson/console/ModelHyperlinkNote.java

```diff
@@ -26,7 +27,7 @@ protected String extraAttributes() {
         return " class='model-link'";
     }
 
-    public static String encodeTo(User u) {
+    public static String encodeTo(@Nonnull User u) {
```

Comment:
Ideally all methods in this class should be verified (potential risk of NPEs, as always)